src/routes/+page.svelte

```diff
@@ -27,8 +27,10 @@
     <button on:click={() => setSquadFilter(3)}>Squad D</button>
     <button on:click={() => setSquadFilter(4)}>Squad E</button>
     <button on:click={() => setSquadFilter(5)}>Squad F</button>
-    <input type="search" bind:value={$searchStore.search} placeholder="Zoek op naam" />
+    <h1>Choose your card</h1>
+    <input class="searchBar" type="search" bind:value={$searchStore.search} placeholder="Zoek op naam" />
 </nav>
+<div class="line"></div>
```

Comment:
Die lijn `<div>` kan ook een `<hr>` zijn, dat is de html tag die daarvoor gemaakt is